spec/linux/kernel.go

```diff
@@ -42,5 +43,72 @@ func (g *KernelGenerator) Generate() (interface{}, error) {
 		results[key] = str
 	}
 
+	hostInfo, err := host.Info()
+	if err != nil {
+		kernelLogger.Errorf("Failed to get platform information: %s", err)
+	}
+
+	if np := normalizePlatform(hostInfo.Platform); np != "" {
+		platformInfo := np + " " + hostInfo.PlatformVersion
+
+		// when platform version is empty string
+		platformInfo = strings.TrimSpace(platformInfo)
+
+		results["platform"] = platformInfo
+	}
+
 	return results, nil
 }
+
+func normalizePlatform(platform string) string {
+	var normalized string
+
+	switch platform {
+	case "debian":
+		normalized = "Debian"
+	case "ubuntu":
+		normalized = "Ubuntu"
+	case "linuxmint":
+		normalized = "Linux Mint"
+	case "raspbian":
+		normalized = "Raspbian"
+	case "fedora":
+		normalized = "Fedora"
+	case "oracle":
+		normalized = "Oracle Linux"
+	case "centos":
+		normalized = "CentOS"
+	case "redhat":
+		normalized = "Red Hat Enterprise Linux"
+	case "scientific":
+		normalized = "Scientific Linux"
+	case "amazon":
+		normalized = "Amazon Linux"
+	case "xenserver":
+		normalized = "XenServer"
+	case "cloudlinux":
+		normalized = "CloudLinux"
+	case "ibm_powerkvm":
+		normalized = "IBM PowerKVM"
+	case "suse":
+		normalized = "SUSE Linux Enterprise Server"
+	case "opensuse":
+		normalized = "openSUSE"
+	case "gentoo":
+		normalized = "Gentoo Linux"
+	case "slackware":
+		normalized = "Slackware"
+	case "arch":
+		normalized = "Arch Linux"
+	case "exherbo":
+		normalized = "Exherbo"
+	case "alpine":
+		normalized = "Alpine Linux"
+	case "coreos":
+		normalized = "CoreOS"
+	default:
+		normalized = ""
```

Comment:
[comment]

```
		platform = strings.ToLower(lsb.ID)
		version = lsb.Release
```
https://github.com/shirou/gopsutil/blob/859c81da50839c6829da58c9e90e1fff1edd8194/host/host_linux.go#L330-L331

platform can be non-empty `lsb.ID` string, so it might be useful not to discard information.

but, i don't know what Isb.ID is like and i assume normalized name should be readable string, so it could be ok as it it.